JokeRandomizer/JokeRandomizer/Views/FirstScreenViewController.swift

```diff
@@ -6,11 +6,19 @@
 //
 
 import UIKit
+import Combine
+import CoreData
 
-class FirstScreen: UIViewController {
+class FirstScreenViewController: UIViewController {
 //MARK: - Property
     private let jokesVM = JokesViewModel()
-    var jokes: Jokes!
+    private let jokesCoreData = JokesDataManager()
```

Comment:
data handling/fetching must be done in viewModel. ViewControllers are used only to display information

Reply:
calling is on view model using init()